src/ContainerAppHelper.ts

```diff
@@ -270,6 +270,38 @@ export class ContainerAppHelper {
         }
     }
 
+    /**
+     * Gets the location of an existing Container App Environment
+     * @param environmentName - the name of the Container App Environment
+     * @param resourceGroup - the resource group that the Container App Environment is found in
+    */
+    public async getExistingContainerAppEnvironmentLocation(environmentName: string, resourceGroup: string) {
+        try {
+            let command = `az containerapp env show -g ${resourceGroup} --query location -n ${environmentName}`;
+            let executionResult = await util.execute(command);
+            return executionResult.exitCode === 0 ? executionResult.stdout.toLowerCase().replace(/["() ]/g, "").trim() : null;
```

Comment:
If I understood properly, this logic is converting the region names from user friendly "North Central US (Stage)" to "northcentralusstage"? Not a blocker for this PR as I believe this will work in practice, but I believe we should be able to pass the user friendly regions as arguments as well, as long as they are between quotes.

Reply:
Just so I'm clear , if it's provided as an argument from the customer I think it will be `northcentralusstage` but if we extracted from the `show` command, afaik we need to convert it to the user friendly format.
Did that answer this question?

Reply:
Honestly I don't think we need to worry too much about this for now, as in practice the logic here should work I expect. My comment was just that potentially we might not even need to convert?
Whatever the format we get to identify the region ("northcentralusstage" or "North Central US (Stage)"), I expect that we should be able to pass this without conversion to the other commands as long as we put the region name between quotes. Again, no need to look into this more for this PR: I think the current approach is OK as well.